### DISCUSSION TITLE
个别视频出现未指定的错误0x80004005

### DISCUSSION THREAD

**Discussion by Playfulnurse** [ywmoyue/biliuwp-lite#177] [General]
![image](https://github.com/ywmoyue/biliuwp-lite/assets/109536052/707dea81-6009-43ea-92f7-faaf4de7d239)

**Comment by ywmoyue** [comment]
系统什么版本

**Reply by Playfulnurse** [reply]
4.6.4.3

**Reply by ywmoyue** [reply]
额，我问的是操作系统版本

**Reply by ywmoyue** [reply]
建议使用脚本进行安装，这里是安装教程：https://github.com/ywmoyue/biliuwp-lite/blob/master/document/install-readme.md
